chord/chord.go

```diff
@@ -0,0 +1,5 @@
+package chord
+
+type Chord struct {
```

Comment:
A.k.a. Gossiper?

Reply:
Can't say for sure. We might need a separate struct for Chord related functionality. Let's keep it as it is for now and then think if we want to be a more general thing or compose it into a separate Peerster struct with other Peerster functionality.